packager/media/event/muxer_listener_internal.cc

```diff
@@ -122,6 +122,23 @@ void AddAudioInfo(const AudioStreamInfo* audio_stream_info,
     }
     audio_info->mutable_codec_specific_data()->set_ec3_channel_map(
```

Comment:
Now that we have more usages of mutable_codec_specific_data, we can create an intermediate variable to make the code more readable, i.e.

```c++
auto* codec_data = audio_info->mutable_codec_specific_data();
codec_data->set_ec3_channel_map(ec3_channel_map);
...
codec_data->set_ec3_channel_mpeg_value(ec3_channel_mpeg_value);
...
```

Reply:
Good suggestion. I will update it.